libs/header/p2p/session.go

```diff
@@ -179,7 +179,7 @@ func (s *session[H]) doRequest(
 			return
 		case s.reqCh <- req:
 		}
-		log.Errorw("processing response", "err", err)
+		log.Errorw("processing response", "from", req.GetOrigin(), "to", req.Amount+req.GetOrigin()-1, "err", err)
```

Comment:
Unrelated to this PR and can extract if reeeealy necessary

Reply:
I'm fine w this